src/test/java/uk/gov/hmcts/reform/ccd/document/am/feign/CaseDocumentClientTest.java

```diff
@@ -85,13 +90,17 @@ void setUp() {
     }
 
     @Test
+    // TODO: fix this test - use other PR to fix
```

Comment:
How do we know when we can remove this TODO - should we reference the PR/JIRA ticket at least ?

Reply:
sorry that was for me only! i needed to delete this one